docs/getting-started/about-mlapi.md

```diff
@@ -3,23 +3,23 @@ id: about-mlapi
 title: Getting Started with MLAPI
```

Comment:
you might want to rename this document too. I'd suggest `about.md` or `about-netcode.md`